pkg/templates/node.go

```diff
@@ -79,7 +81,7 @@ systemd:
           --cluster-dns={{ .ClusterDNSAddress }} \
           --cluster-domain={{ .ClusterDomain }} \
           --client-ca-file=/etc/kubernetes/certs/kubelet-clients-ca.pem \
-          --non-masquerade-cidr={{ .ClusterCIDR }} \
+          --non-masquerade-cidr=0.0.0.0/0 \
```

Comment:
I think we can remove the setting completely. Should have the same effect.

Reply:
I stand corrected. The default seems to be `10.0.0.0/8` which would not be what we want. Setting it to `0.0.0.0/0` seems to be the way to go.